php1/mysql/reset/otp.php

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>OTP</title>
+  <link rel="stylesheet" type="text/css" href="../../css/style1.css">
+  <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.6.3/jquery.min.js"></script>
+</head>
+<body>
+  <?php require 'otpvalidate.php'; ?>
+
+  <div class="container">
+    <div class="login">
+      <h1 class="head">Enter the OTP</h1>
+      <form action="otp.php" method="post">
+        <span class="error" id="msg"><?php echo $error; ?></span><br>
```

Comment:
I think in `<span>` which `id` is `msg` you are printing error message so please add condition when error will come then only `echo` statement should be execute.

Reply:
Actually I'm talking about `echo` statement which is currently executed always irrespective of condition, if there is no error then `$error` varaible is empty but `echo` statement should not be executed this is also discussed by sir that's why I am telling.